### DISCUSSION TITLE
Alboran sea sulation

### DISCUSSION THREAD

**Discussion by ggiulia00** [Parcels-code/Parcels#1777] [Q&A]
### Question

Hi everyone!
i'm very new in this stuff so i'm a bit struggling in doing it.
I have to simulate the inflow of microplastics from rivers (Gaudiaro, Gaudalfeo, Guadalhorce, Moulouya and Tafna) in the alboran sea. the simulation should start on 3rd November 2023 for 10 days.
i tryed to download a nc file from copernicus with hydrodynamic date but when I try to open it on python script it fails.
surely i do something wrong but i really don't understand.

thanks
Giulia

**Comment by VeckoTheGecko** [comment]
Hi Giulia,
Have you had a look through [our tutorials](https://docs.oceanparcels.org/en/latest/documentation/index.html), and tried adapting code from there? The tutorials are a great starting place for beginner Parcels users.

If after trying that you're still stuck, you're welcome to provide a sample of your code and we can have a look.

**Reply by ggiulia00** [reply]
yes, I do but i don't understand how to simulate what i need, maybe you can suggest what to do
i'm sorry for theese stupid and simple question but i've never worked in stuff like this

**Reply by ggiulia00** [reply]
i don't know, maybe the problem is the nc file because the program can't read it

**Reply by VeckoTheGecko** [reply]
That's all good. Can you provide your code here? You can paste it in a [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks)

**Reply by ggiulia00** [reply]
I'm going to try now!

**Reply by ggiulia00** [reply]
finally I couldn't do anything, i'm sorry. i'm trying againg but i always fail
